### PR TITLE
Update table, page and layout styles

### PR DESCRIPTION
**Pull request by LanesGood**
This PR makes a number of small style changes that overall have a larger effect. I began looking at responsive styles for tables and pages, and then moved to making styles more consistent across the board, and then ended up making some more changes.

Screenshots:
image

image
image
image
image

### CODE REVIEW

File: src/components/edit-team-form.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ export default function EditTeamForm({
               <div className='form-control form-control__vertical'>
                 <label htmlFor='organization'>Add to Organization</label>
                 <Field as='select' name='organization'>
-                  <option value=''>No organization</option>
+                  <option value=''>Select organization</option>
```
Comment (vgeorge) [inline src/components/edit-team-form.js:175]: @LanesGood this suggests that selecting an organization is required, which is not the case. The current option value ("No organization") is also unclear. One possible solution is to add a checkbox to the Add to organization" label and display the dropdown only when it is selected.

**Review by vgeorge — CHANGES_REQUESTED** [review]
@LanesGood styles looks great. I left a comment about a change in the team creation page.

File: src/components/edit-team-form.js (hunk 1 above)
Comment (LanesGood) [reply src/components/edit-team-form.js:175]: @vgeorge I agree, this flow isn't so clear. I like your suggestion - I'll see if I can implement this.

Comment (LanesGood) [reply src/components/edit-team-form.js]: Updated flow: 
https://user-images.githubusercontent.com/12634024/213499166-30d3fa92-2d6e-464b-85a4-455f95b29a9b.mov

**Review by vgeorge — APPROVED** [review]

**Pull request merged**